interface/forms/eye_mag/SpectacleRx.php

```diff
@@ -920,90 +855,94 @@ function reverse_cylinder() {
+                                        </span>
+                                        <br/>
+                                        <span id="Trifocal_span" name="Trifocal_span">
+                                            <label for="RXTYPE_Trifocal"><?php echo xlt('Trifocal'); ?></label>
                                             <input type="radio"
                                                    onclick="pick_rxType('Trifocal',<?php echo attr_js($insert_this_id); ?>);"
                                                    value="Trifocal" id="RXTYPE_Trifocal"
-                                                   name="RXTYPE" <?php echo attr($Trifocal); ?>></b><br/>
-                                        <b id="Progressive_span">
+                                                   name="RXTYPE" <?php echo attr($Trifocal); ?>>
+                                        </span>
+                                        <br/>
+                                        <span id="Progressive_span">
                                             <label for="RXTYPE_Progressive">
                                                 <?php echo xlt('Prog.{{Progressive lenses}}'); ?>
                                             </label>
                                             <input type="radio"
                                                    onclick="pick_rxType('Progressive',<?php echo attr_js($insert_this_id); ?>);"
                                                    value="Progressive" id="RXTYPE_Progressive"
-                                                   name="RXTYPE" <?php echo attr($Progressive); ?>></b><br/>
+                                                   name="RXTYPE" <?php echo attr($Progressive); ?>>
+                                        </span>
+                                        <br/>
                                     </td>
                                     <td></td>
                                 </tr>
                                 <tr class="center">
-                                    <td rowspan="2" colspan="1" class="right bold"><?php echo xlt('Distance'); ?>: </td>
-                                    <td class="right bold"><?php echo xlt('OD{{right eye}}'); ?></td>
+                                    <td rowspan="2" colspan="1" class="right font-weight-bold"><?php echo xlt('Distance'); ?>: </td>
+                                    <td class="right font-weight-bold"><?php echo xlt('OD{{right eye}}'); ?></td>
                                     <td><input type=text id="ODSPH" name="ODSPH" value="<?php echo attr($ODSPH); ?>"></td>
                                     <td><input type=text id="ODCYL" name="ODCYL" value="<?php echo attr($ODCYL); ?>"></td>
                                     <td><input type=text id="ODAXIS" name="ODAXIS" value="<?php echo attr($ODAXIS); ?>">
```

Comment:
This type attribute should have quotes surrounding it -- seems this file needs to be inspected for that sorta stuff.

Reply:
I am not sure your mean, Can you describe more detail?